### PR TITLE
chore(config): set token decimals

### PR DESCRIPTION
**Pull request by enddynayn**
Set token decimals to 8.

Part of #263

### CODE REVIEW

**Review by saraswatpuneet — APPROVED** [review]

File: node/service/src/chain_spec/frequency_local.rs [hunk 1]
```diff
@@ -23,7 +23,7 @@ pub fn template_session_keys(keys: AuraId) -> frequency_rococo_runtime::SessionK
 
 pub fn local_testnet_config() -> ChainSpec {
 	// Give your base currency a unit name and decimal places
-	let properties = get_properties("UNIT", 12, 42);
```
Comment (saraswatpuneet) [inline node/service/src/chain_spec/frequency_local.rs:26]: Yes this is the :snake:

Comment (enddynayn) [reply node/service/src/chain_spec/frequency_local.rs:26]: lol

**Review by wilwade — APPROVED** [review]

File: node/service/src/chain_spec/frequency_local.rs (hunk 1 above)
Comment (shannonwells) [reply node/service/src/chain_spec/frequency_local.rs:26]: SSSSSSS

**Review by shannonwells — APPROVED** [review]
🚢 it

**Pull request merged**